app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```

Comment:
- [ ] An error occurs when I attempt to sign up. This is because `name` field is required in the sign up process, but you haven't made this provision. Kindly see error below 👇 
![image](https://user-images.githubusercontent.com/46839548/181051741-fc478c38-07c7-4dd6-9538-412541963406.png)